scripts/create-family-discussions.py

```
@@ -0,0 +1,148 @@
+    "2024 XMR SimpleX": ["2024 XMR SimpleX"],
+    "3a Rule": ["3a Rule"],
+}
+
+headers = {
+    "Authorization": f"Bearer {TOKEN}",
+    "Content-Type": "application/json",
+    "Accept": "application/vnd.github+json"
+}
+
+# Check existing
+resp = requests.post(API_URL, headers=headers, json={
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)